packages/contracts/src/plugins/placeholder-version/PlaceholderSetup.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.8.17;
+
+import {PermissionLib} from "../../core/permission/PermissionLib.sol";
+import {PluginSetup, IPluginSetup} from "../../framework/plugin/setup/PluginSetup.sol";
+
+/// @title PlaceholderSetup
+/// @author Aragon Association - 2023
+/// @notice The setup contract acting as a placeholder for outdated plugin builds. These are used to save gas when giving builds and releases identical plugin repos after deployment that do not need to be published when deploying a plugin repo to a new chain or layer.
```

Comment:
it doesn't really save gas but prevent users from installing an outdated plugin version

Reply:
Fixed in 95d57e1695705286cec906153defb7a40e06f408